db/rdbms/src/main/java/io/camunda/db/rdbms/write/domain/DecisionDefinitionDbModel.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.write.domain;
+
+import io.camunda.util.ObjectBuilder;
+
+public record DecisionDefinitionDbModel(
+    Long decisionDefinitionKey,
```

Comment:
❓ Wouldn't be better to use primitives? I think the exporter mostly receives primitive values. Or does MyBatis needs them?

Reply:
The DecisionDefinitionEntity also uses the Wrapper Type. Since we also use Wrapper Types for all other DbModels, I would suggest too keep it as it is.

Reply:
Also when using constructor mapping in the resultMap, primitives don't work. So I'd like to keep it the same everywhere